Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue5555.cs

```diff
@@ -59,6 +59,8 @@ protected override void Init ()
 						result.Text = "Click 'Push page' again";
 						return;
 					}
+					GC.Collect();
+					GC.WaitForPendingFinalizers();
```

Comment:
In general, we should call `GC.Collect();` after waiting for pending finalizers so that we can collect any dead objects which are created when calling the finalizers, so I
d add another `GC.Collect();` here..